### ISSUE TITLE
Label suggestion: algorithm / algorithms

### ISSUE THREAD

**Issue by mudit458** [despo/issuehub.io#315]

**Comment by xanderyzwich** [comment]
I don't believe that this label would be helpful in a real world project. All code uses algorithms, but issues don't typically center around this subject. "optimization" and "refactoring" would likely cover this already.
